### DISCUSSION TITLE
Welcome to Express-HTMX-Template Discussions!

### DISCUSSION THREAD

**Discussion by heysubinoy** [subinoybiswas/Express-HTMX-Template#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kasir-barati** [comment]
Hey, I am Mohmmad Jawad. Being a fullstack dev for about a year I was wondering what is this htmx and how can I get a simple todo app working. So I embarked on a journey to the unknown. So far I read your codebase and to say the least it is fine but needs some polishing.

You know what I mean, ExpressJS v5 was released just a couple of days ago. You no longer need `body-parser` in your ExpressJS app: `express.json()` will do the trick. And most importantly, you need to show how people can send dynamic data, but now all I've ever seen is reading an HTML file and sending it back. It is fine but I guess we need to think about it from the perspective that we have a db (although we can ignore a real db and just read some dummy data exported from 'dummy-data.ts').

So here I am trying to address these things :slightly_smiling_face:.
